mitreattack/navlayers/core/README.md

```diff
@@ -0,0 +1,245 @@
+# Layers Core
+This subcomponent, as part of the larger navlayers module, is responsible for Layer objects. Please note, this 
+documentation assumes familiarity with the [ATT&CK Navigator layer format](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv4_1.md).
+The main handle for this implementation is the Layer, which stores an individual instance of a LayerObj object,
+which further references the various sub-objects that make up a complete Layer. A visual representation of this
+object breakdown can be seen here (please note there are other fields, these are just the objects):
+```
+demo (Layer instance) <------------------------------------------------> The container for a layer object
+  |---> demo.layer (_LayerObj instance)--------------------------------> The raw layer object itself
+          |---> demo.layer.version (Versions instance)-----------------> A versions object
+          |---> demo.layer.filters (Filter instance)-------------------> A filter object
+          |---> demo.layer.layout (Layout instance)--------------------> A layout object
+          |---> demo.layer.techniques (List of Technique instances)----> A collection of technique objects
+          |---> demo.layer.gradient (Gradient instance)----------------> A gradient object
+          |---> demo.layer.legendItems (List of LegendItem instances)--> A collection of legend item objects
+          |---> demo.layer.metadata (List of Metadata instances)-------> A collection of metadata objects
+```
```

Comment:
Super useful graphic, glad you included it.

Reply:
It was a bit of a pain to make, but it's so easy to tell what's going on with it that the effort was absolutely worth it.